streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -1643,6 +1663,8 @@ public void process(final String key, final String value) {
                     new KeyValue<>("A", "recurse-alpha")
                 ))
             );
+
+            assertThat(topologyTestDriver.producedTopicNames(), hasItem("globule-topic"));
```

Comment:
IMHO, this should be it's own test similar to the one from above. (Btw: what is "globule" -> should we fix those names?)

Reply:
You're right, I was being lazy ;)

Reply:
There is no global table changelog topic -- this test really only write into the global input topic and your verification step was to verify that this write is picked up. Will add a corresponding test in a follow up PR.